helm-charts/seldon-core-analytics/README.md

```diff
@@ -4,24 +4,24 @@ This is a Prometheus and Grafana installation with a basic Grafana dashboard sho
 
 ## Installation
 
-The Helm chart takes the following parameters:
-
-  * ```grafana_prom_admin_password``` : The password for logging into Grafana with the admin account
-  * ```persistence.enabled``` : Whether to try and run with persistence. If ```true``` then you need to provide a persistent volume with a claim name ```seldon-claim``` which Prometheus will mount to store metrics.
-
+The Helm chart parameters for Prometheus and Grafana can be found and edited in values.yaml, examples of which can be seen for Prometheus [here](https://github.com/helm/charts/blob/master/stable/prometheus/values.yaml) and for Grafana [here](https://github.com/helm/charts/blob/master/stable/grafana/values.yaml).
 
 An example install is shown below:
 
 ```
-helm install seldon-core-analytics --name seldon-core-analytics --set grafana_prom_admin_password=password --set persistence.enabled=false --repo https://storage.googleapis.com/seldon-charts --namespace seldon
+kubectl create namespace seldon
+```
+
+```
+helm install seldon-core-analytics . -n seldon
```

Comment:
Can we install this into the `seldon-system` namespace by default? Thoughts?

Reply:
Yes I think we should

Reply:
Fixed